reg_tests/grid_gen/driver.hera.sh

```diff
@@ -86,27 +86,36 @@ TEST3=$(sbatch --parsable --ntasks-per-node=24 --nodes=1 -t 0:07:00 -A $PROJECT_
       -o $LOG_FILE3 -e $LOG_FILE3 ./gfdl.regional.sh)
 
 #-----------------------------------------------------------------------------
-# esg regional grid
+# ESG regional grid (output dominate soil/vegetation type).
```

Comment:
Dominant?  Same comment for change below in this file and in other files.

Reply:
Thank you. I can't spell.